source/features/load-hidden-comments.tsx

```diff
@@ -0,0 +1,50 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	// Retrieve the form associated with the "X Hidden Items\nLoad More" button
+	const form = select('form.ajax-pagination-form');
+	if (form) {
```

Comment:
Skip this, look for the button directly, with a specific selector.

Later you can find the form via `button.form!`